pkg/plugins/resources/yaml/main.go

```diff
@@ -8,24 +8,24 @@ import (
 
 	"github.com/mitchellh/mapstructure"
 	"github.com/sirupsen/logrus"
+	"github.com/updatecli/updatecli/pkg/core/result"
 	"github.com/updatecli/updatecli/pkg/core/text"
 
 	"gopkg.in/yaml.v3"
 )
 
-var (
-	yamlIdent int = 2
-)
-
 // Spec defines a specification for a "yaml" resource
 // parsed from an updatecli manifest file
 type Spec struct {
-	// Specifies the YAML file
-	File string `yaml:",omitempty"`
-	// Specifies the YAML key
-	Key string `yaml:",omitempty"`
-	// Specifies the YAML value
-	Value string `yaml:",omitempty"`
+	// [source,condition,target] File contains the file path to take in account
+	File string
```

Comment:
```suggestion
	File string `yaml:",omitempty"`
```

Reply:
Please not that linting may fail because of the wrong space introduce between File string and `yaml:",omitempty"`